### PR TITLE
Add missing import

### PR DESCRIPTION
**Pull request by ivanjeremic**

### CODE REVIEW

File: docs/stories/components/TableComplex.stories.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { sortBy as sort } from 'lodash';
 import { storiesOf } from '@storybook/react';
 
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faTrashAlt, faPencilAlt } from '@fortawesome/free-solid-svg-icons';
+import { faPencilAltl, faTrashAlt, faPencilAlt } from '@fortawesome/free-solid-svg-icons';
```
Comment (soupette) [inline docs/stories/components/TableComplex.stories.js:10]: The imports are correct however it's the example that is wrong can you correct it line 391?

**Comment by soupette** [comment]
@ivanjeremic  can you make sure the DCO checks so we can merge your PR?

**Comment by ivanjeremic** [comment]
> @ivanjeremic can you make sure the DCO checks so we can merge your PR?

I did what is described here, [https://github.com/strapi/buffet/pull/151/checks?check_run_id=928038581](url) It is the first time I try to contribute somewhere, hope you can help me find the reason it is still blocked, I did `git rebase HEAD~2 --signoff´ in my local copy and then `git push --force-with-lease origin master` because `patch-1´ branche was gone.

**Comment by ivanjeremic** [comment]
> @ivanjeremic can you make sure the DCO checks so we can merge your PR?

Done :)

**Comment by soupette** [comment]
@ivanjeremic there's no file changed anymore in your PR

**Comment by ivanjeremic** [comment]
> @ivanjeremic there's no file changed anymore in your PR

It's the first time I try to contribute anywhere, what exactly did I wrong or missed? Thanks

**Comment by soupette** [comment]
It appears another PR already fixed the missing import.

I am closing your PR then, feel free to reopen a new one! 

Thanks again for taking the time to contribute!

**Pull request closed without merging**